src/WebJobs.Extensions.NotificationHubs/Config/NotificationHubsJobHostConfigurationExtensions.cs

```diff
@@ -0,0 +1,32 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System;
+
+namespace Microsoft.Azure.WebJobs.Extensions.NotificationHubs
```

Comment:
This needs to be in the root Microsoft.Azure.WebJobs namespace like the other extensions. We want it to be immediately in scope for people when they add the package.

Generally for all extensions I write I do an e2e trial in a new project, referencing the package, and verifying everything lights up the way it should.